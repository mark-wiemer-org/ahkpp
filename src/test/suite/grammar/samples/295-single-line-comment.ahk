; [Issue #295](https://github.com/mark-wiemer/vscode-autohotkey-plus-plus/issues/295)
ToolTip
ToolTip ;Comment
ToolTip;NotComment
OpenLabel:
OpenLabel: ;Comment
OpenLabel:;NotComment
#KeyHistory
#KeyHistory MaxEvents
#KeyHistory MaxEvents ;Comment
#KeyHistory MaxEvents;NotComment
