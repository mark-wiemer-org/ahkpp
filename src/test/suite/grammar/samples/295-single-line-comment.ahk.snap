>ToolTip;NotComment
#^^^^^^^ source.ahk support.function.ahk
#       ^ source.ahk
#        ^^^^^^^^^^ source.ahk variable.def.ahk
>ToolTip ;Comment
#^^^^^^^ source.ahk support.function.ahk
#       ^ source.ahk
#        ^ source.ahk comment.line.semicolon.ahk punctuation.definition.comment.ahk
#         ^^^^^^^ source.ahk comment.line.semicolon.ahk
>